sdk/keyvault/test-resources.bicep

```diff
@@ -0,0 +1,205 @@
+@allowed([
```

Comment:
nit: maybe we remove the `allowed` values here - it probably just adds confusion more than anything else. Feel free to ignore since this is already present in the arm template

Reply:
Done.